drivers/adodb-postgres64.inc.php

```diff
@@ -728,11 +728,9 @@ function _connect($str,$user='',$pwd='',$db='',$ctype=0)
 		if ($this->_connectionID === false) return false;
 		$this->Execute("set datestyle='ISO'");
 
-		$info = $this->ServerInfo();
```

Comment:
$info is also used at line 741 below, so removing this line requires additional changes to ensure the version check works.